src/Search/Search.php

```diff
@@ -297,9 +357,7 @@ public function buildBody()
             $from = ($this->getPage() - 1) * $this->getSize();
         }
 
-        if ($from > 0) {
-            $body['from'] = $from;
-        }
+        $body['from'] = $from;
```

Comment:
This was done on purpose, I wouldn't remove it

Reply:
I copied it from an another version of our package (directly from my working vendor/lumen-elasticsearch). I will fix these differences.